dashboard/config/scripts/poorva_match_pl.match

```diff
@@ -6,6 +6,6 @@ answer 'red_sprite rotates counter clockwise'
 question 'Press the w key'
 answer 'blue_sprite rotates clockwise'
 question 'Press the j key'
-answer 'red_sprite moves up and blue_sprite moves down'
+answer 'red\_sprite moves up and blue\_sprite moves down'
```

Comment:
Do I also need to worry about other uses of `_` such as this one? https://github.com/code-dot-org/code-dot-org/blob/c2c28e254879a3e87b15ad1f4d93e596ee6bcf46/dashboard/config/scripts/u1l13_assess_text_compression_reverse_process_2018.text_match#L3

Reply:
Nope! That's a text match level; we don't render that string, we just use it to check the entered answer